portfolio/src/main/java/com/google/sps/data/Quote.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+/** Class that models a quote. */
+public final class Quote {
+
+  private final long id;
+  private final String text;
+  private final long timestamp;
```

Comment:
What is the unit of this timestamp? Consider updating the variable name, here and elsewhere.

Ex:
timestampMillis

Reply:
Done.